gallery_dl/extractor/instagram.py

```diff
@@ -86,7 +102,8 @@ def _extract_profilepage(self, url):
 
             for s in shortcodes:
                 url = '{}/p/{}/'.format(self.root, s)
-                yield self._extract_postpage(url)
+                for p in self._extract_postpage(url):
+                    yield p
```

Comment:
`yield from self._extract_postpage(url)`

Reply:
Nice catch, thanks! (I have just committed it!)